### PR TITLE
Updated DataGrid FIltering topics

### PR DESCRIPTION
**Pull request by kerpecheva**

### CODE REVIEW

File: controls/datagrid/filtering/filter-control-template.md [hunk 1]
```diff
@@ -14,6 +14,46 @@ In addition, as the [template column]({%slug datagrid-columns-template-column%})
 
 * `FilterControlTemplate`(`DataTemplate`)&mdash;Specifies the user defined template used for Filtering UI. The template must contain an instance of the `Telerik.Maui.Controls.Compatibility.DataGrid.DataGridFilterControlBase` class. 
 
+The DataGrid provides a few predefined filter controls which are applied depending on the column type:
```
Comment (ikoevska) [inline controls/datagrid/filtering/filter-control-template.md:17]:
```suggestion
The DataGrid provides the following predefined filter controls which are applied depending on the column type:
```

File: controls/datagrid/filtering/filter-control-template.md [hunk 2]
```diff
@@ -14,6 +14,46 @@ In addition, as the [template column]({%slug datagrid-columns-template-column%})
...
+* `DataGridTextFilterControl`
+* `DataGridNumericFilterControl`
+* `DataGridBooleanFilterControl`
+* `DataGridDateTimeFilterControl`
+* `DataGridTimeFilterControl`
+* `DataGridComboBoxFilterControl`
+
+The `FilterControlTemplate` property of the columns allows you to customize its corresponding filter control. If the default filter control does not meet the requirements, you can create a custom filter control from scratch. 
+
+## Example with modified TextFilterControl
```
Comment (ikoevska) [inline controls/datagrid/filtering/filter-control-template.md:28]:
```suggestion
## Example with Modified TextFilterControl
```

File: controls/datagrid/filtering/filter-control-template.md [hunk 3]
```diff
@@ -14,6 +14,46 @@ In addition, as the [template column]({%slug datagrid-columns-template-column%})
...
+
+The example below shows how to apply the `DataGridTextFilterControl` as a `FilterControlTemplate` of the text column and set the filter match case options to `False`. This means the toggle buttons inside the filter template for switching to case-sensitive filter will be untoggled when the filtering UI appears.
```
Comment (ikoevska) [inline controls/datagrid/filtering/filter-control-template.md:30]:
```suggestion
The example below shows how to apply the `DataGridTextFilterControl` as a `FilterControlTemplate` of the text column and set the filter match case options to `False`. This means that the toggle buttons inside the filter template for switching to case-sensitive filter will be untoggled when the filtering UI appears.
```

File: controls/datagrid/filtering/filtering-ui.md [hunk 4]
```diff
@@ -8,20 +8,21 @@ slug: datagrid-filtering-ui
 
 # Filtering UI
 
-The [.NET MAUI DataGrid]({%slug datagrid-overview%}) supports filtering through the UI.
+The [.NET MAUI DataGrid]({%slug datagrid-overview%}) provides you with a built-in filtering functionality, which allows the user to filter the data by one or more columns by clicking on the filtering icon next to the header text.
 
 ![Filtering UI](../filtering/images/datagrid-filtering-ui.gif)
 
-> Filtering UI appears when clicking on the filtering icon on each column's header'.
+> Filtering UI appears when clicking on the filtering icon on each column's header.
```
Comment (ikoevska) [inline controls/datagrid/filtering/filtering-ui.md:15]:
```suggestion
> The filtering UI appears when clicking the filtering icon in each column's header.
```

File: controls/datagrid/filtering/filtering-ui.md [hunk 5]
```diff
@@ -8,20 +8,21 @@ slug: datagrid-filtering-ui
...
 
+You can control whether the filtering UI is available to the end users for the whole DataGrid with the `UserFilterMode` property:
 
-The Filtering UI exposes the following properties:
+* `UserFilterMode`(`DataGridUserFilterMode`)&mdash;Defines whether the Filter Glyph is present in the header of the currently available filterable columns. The available options are `Auto`, `Enabled`, and `Disabled`. The default value of the `UserFilterMode` is `Auto`.
```
Comment (ikoevska) [inline controls/datagrid/filtering/filtering-ui.md:19]: Is the "Filter Glyph" the same as the "filtering icon"?

File: controls/datagrid/filtering/filtering-ui.md [hunk 6]
```diff
@@ -8,20 +8,21 @@ slug: datagrid-filtering-ui
...
 
-* `UserFilterMode`&mdash;Defines whether the Filtering UI is enabled/disabled. The available options are `Auto`, `Enabled`, and `Disabled`. The default value of the `UserFilterMode` is `Auto`. Set the property on DataGrid control.
+The DataGrid columns expose the following properties related to the filtering UI: 
 
-* `CanUserFilter`(`bool`) property is used to enable/disable the filtering of a specific column. Set the property to a specific column.
+* `CanUserFilter`(`bool`)&mdash;Indicates whether the user can filter this column by using the built-in filtering UI. By default, the value is `True`.
 
-* `ShowDistinctValuesFilter`(`bool`)&mdash;Defines whether the distinct values are included in the Filtering UI. By default the value is true.
+* `ShowDistinctValuesFilter`(`bool`)&mdash;Defines whether the distinct values are included in the Filtering UI. By default, the value is `True`.
```
Comment (ikoevska) [inline controls/datagrid/filtering/filtering-ui.md:25]:
```suggestion
* `ShowDistinctValuesFilter`(`bool`)&mdash;Defines whether the distinct values are included in the filtering UI. By default, the value is `True`.
```

File: controls/datagrid/filtering/overview.md [hunk 7]
```diff
@@ -11,20 +11,30 @@ slug: datagrid-filtering-overview
 
 The [Telerik UI for .NET MAUI DataGrid]({%slug datagrid-overview%}) supports filtering operations either through the UI - using the Filtering UI or programmatically.
 
-![Filtering UI](../filtering/images/datagrid-filtering-ui.gif)
-
 ## Filtering UI
 
 > Filtering UI appears when clicking on the filtering icon on each column's header'.
```
Comment (ikoevska) [inline controls/datagrid/filtering/overview.md:16]:
```suggestion
> The filtering UI appears when clicking the filtering icon in each column's header.
```

File: controls/datagrid/filtering/overview.md [hunk 8]
```diff
@@ -11,20 +11,30 @@ slug: datagrid-filtering-overview
...
 
-## FilterControl Template
+![Telerik .NET MAUI DataGrid Filtering UI](../filtering/images/datagrid-filteringui-platforms.png)
+
+## Filter Control Template
 
-The Telerik DataGrid allows you to apply filtering to the DataGrid column using the `FilterControlTemplate` property.
+The Telerik DataGrid allows you to apply custom filter control to the DataGrid column using the `FilterControlTemplate` property.
```
Comment (ikoevska) [inline controls/datagrid/filtering/overview.md:22]: Would "custom filter controls" make more sense here?

File: controls/datagrid/filtering/overview.md [hunk 9]
```diff
@@ -11,20 +11,30 @@ slug: datagrid-filtering-overview
...
 
 * `FilterControlTemplate`(`DataTemplate`)&mdash;Specifies the user defined template used for Filtering UI. The template must contain an instance of the `Telerik.Maui.Controls.Compatibility.DataGrid.DataGridFilterControlBase` class.
 
+>Go to [Filter Control Template]({%slug datagrid-filter-control-template%}) topic for detailed information on how to create a custom filter control.
+
 ## Programmatic Filtering
 
 Programmatic filtering is achieved by adding different filter descriptors in the `FilterDescriptor` collection of the control.
 
+>Go to [Programmatic Filtering]({%slug datagrid-programmatic-filtering%}) for detailed information on the provided filter descriptors.
```
Comment (ikoevska) [inline controls/datagrid/filtering/overview.md:32]:
```suggestion
>tip Go to the [Programmatic Filtering]({%slug datagrid-programmatic-filtering%}) topic for detailed information about the provided filter descriptors.
```

**Review by ikoevska — APPROVED** [review]
Approved after the necessary changes are applied.

Note that you're switching between `Filtering UI` and `filtering UI`. I would probably go with `filtering UI` because it's more generic. However, if you'd like to use it as a feature name, you can go with `Filtering UI` throughout.

File: controls/datagrid/filtering/filtering-ui.md (hunk 5 above)
Comment (kerpecheva) [reply controls/datagrid/filtering/filtering-ui.md:19]: Yes, it's the same, I'll make it filtering icon

**Comment by kerpecheva** [comment]
I left it as Filtering UI as it's a feature name

**Pull request merged**